### PR TITLE
maint: update theme version, actions and logo

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**

### CODE REVIEW

**Review by RobPasMue — APPROVED** [review]
LGTM!

File: pyproject.toml [hunk 1]
```diff
@@ -42,7 +42,6 @@ maintainers = [
 classifiers = [
     "Development Status :: 4 - Beta",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
```
Comment (SMoraisAnsys) [inline pyproject.toml:45]: Shouldn't 3.9 be also removed by default (cf https://scientific-python.org/specs/spec-0000/)

**Review by SMoraisAnsys — APPROVED** [review]
LGTM, I left a minor comment

File: pyproject.toml (hunk 1 above)
Comment (RobPasMue) [reply pyproject.toml:45]: I agree but I think we should do that cleanup in a different PR

Comment (SMoraisAnsys) [reply pyproject.toml:45]: Fine by me :)

**Pull request merged**